Tools/eslint-config-cesium/node.js

```diff
@@ -0,0 +1,13 @@
+/*eslint-env node*/
+'use strict';
+
+module.exports = {
+    "extends": "cesium",
```

Comment:
it looks very strange to have only those rules have single-quotes--should we just always quote property names for rules?